tests/onnxruntime/test_optimization.py

```diff
@@ -43,6 +37,11 @@
 from optimum.onnxruntime.modeling_decoder import ORTModelForCausalLM
 from optimum.onnxruntime.modeling_seq2seq import ORTModelForSeq2SeqLM, ORTModelForSpeechSeq2Seq
 from optimum.utils.testing_utils import grid_parameters
+from parameterized import parameterized
+from transformers import AutoTokenizer
+from transformers.onnx.utils import get_preprocessor
+from transformers.testing_utils import require_torch_gpu
+from utils_onnxruntime_tests import MODEL_NAMES
```

Comment:
I ran black— I can undo this part

Reply:
use the library's styling with `make style` after `pip install -e .[quality]`